entrypoint.sh

```diff
@@ -2,6 +2,14 @@
 
 set -o errexit
 
+# export custom environment variables
+ENV_FILE=/srv/grafana/envs
+if [[ -f "$ENV_FILE" ]]; then
+    set -a # enable export for all variables
+    source $ENV_FILE
+    set +a
+fi
+
```

Comment:
@nikita-b It looks like this section should go after `pmm-managed-init`, since that one will panic if it discovers a variable which it doesn't recognize as valid. And all those grafana-specific env vars will be foreign to it...

Reply:
It's a good question actually!
We have exceptions for Grafana variables: 
https://github.com/percona/pmm-managed/blob/PMM-2.0/utils/envvars/parser.go#L162
For all variables starts with `GF_`.
And I had a lot of doubt about the position of these strings (after/before `pmm-managed-init`) but because we can export any variables from envs file, I believe it'll be better to place before checking.

Reply:
Let's do the following:
1/ put this new section after pmm-managed-init and 
2/ not limit the users passing grafana variables only, but any arbitrary variables

Reply:
Looks good, thanks!
@michaelcoburn PTAL as well. Would it solve it?